crossChain.json

```diff
@@ -405,6 +405,12 @@
         "mainnetEquivalent": "0x0100546F2cD4C9D97f798fFC9755E47865FF7Ee6",
         "decimals": 18
       },
+      "0x303241e2B3b4aeD0bb0F8623e7442368FED8Faf3": {
+        "name": "Governance Alcx",
```

Comment:
Are we meaning to add GALCX here and set it up as an xERC20 as well? If so, we'll need the related PR in the monorepo config.